packages/vm/src/index.ts

```diff
@@ -176,7 +177,7 @@ export default class VM extends AsyncEventEmitter {
 
   protected readonly _opts: VMOpts
   protected _isInitialized: boolean = false
-  protected readonly _allowUnlimitedContractSize: boolean
+  public readonly _allowUnlimitedContractSize: boolean
```

Comment:
This is changed to `public` because it is accessed here:

https://github.com/ethereumjs/ethereumjs-monorepo/blob/7330c4ace495903748c606a47a8b993812504db8/packages/vm/src/evm/evm.ts#L428

`DEBUG` is accessed here:

https://github.com/ethereumjs/ethereumjs-monorepo/blob/7330c4ace495903748c606a47a8b993812504db8/packages/vm/src/evm/evm.ts#L409

This is because in the constructor to the `EVM` the first arg is updated from `any` to `VM` here.
https://github.com/ethereumjs/ethereumjs-monorepo/pull/1768/files#diff-44f5139e1722034ba835a86c3333b7452e621e5ecb324b0b12e9c7798f678acfR145

Happy to revert that change if you prefer

Reply:
I feel like its unrelated changes for this PR but probably better to have the argument be type but happy to do whatever you think is best

Reply:
Oh I see, yes much better to type it as VM and get all the type goodness :) this is fine then, thanks!